app/models/partnering_organization.rb

```diff
@@ -1,2 +1,16 @@
+# == Schema Information
+#
+# Table name: partnering_organization
+#
+# name           :string
+# phone_number   :string
+# address        :string
+# latitude       :float
+# longitude      :float
+# category       :integer
+# demographic    :integer
+#
 class PartneringOrganization < ApplicationRecord
+	enum category: [:caregiver, :support_team, :educator, :volunteer_organization, :awareness_team]
```

Comment:
can you change the name of this field from category to "role" so we don't confuse it with the resource category :)

Reply:
also add a has_many relationship to resources